yarn.lock

```diff
@@ -5,14 +5,12 @@
 "@babel/code-frame@^7.0.0":
   version "7.0.0"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
-  integrity sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==
```

Comment:
Which version of Yarn are you using? Looks like it's removing all of these lines.

Reply:
I'm on yarn version `1.9.4`

Reply:
Could you maybe try updating to the latest version of Yarn and regenerating the lockfile?

Reply:
Upgraded to `yarn 1.12.3` and regenerated the `lockfile`